dev-python/python-docs/python-docs-2.7.12.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+DESCRIPTION="HTML documentation for Python"
+HOMEPAGE="https://www.python.org/doc/"
+SRC_URI="https://docs.python.org/2/archives/python-${PV}-docs-html.tar.bz2"
+
+LICENSE="PSF-2"
+SLOT="2.7"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~sparc-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
+IUSE=""
+
+DEPEND=""
+RDEPEND=""
+
+S="${WORKDIR}/python-${PV}-docs-html"
+
+src_install() {
+	dohtml -A xml -A inv -r ./
```

Comment:
Would be nice if you gave it a run with:

    PORTAGE_DOHTML_WARN_ON_SKIPPED_FILES=yes

in make.conf and seen if Portage complains about some files being skipped. If yes, paste them and we'll consider if they should be skipped or installed.

Then you'd need to replace `dohtml` with `docinto` + `dodoc`.